### DISCUSSION TITLE
Memory Leak...?

### DISCUSSION THREAD

**Discussion by nomatterjun** [Favor-Gift-Reminder/Favor-iOS#100] [General]
<img width="1009" alt="스크린샷 2023-03-26 17 07 36" src="https://user-images.githubusercontent.com/60438045/227765143-026bd743-1e18-4dac-bdbc-02c0d2fb5399.png">

`Fetcher`를 계속 돌리면서 관찰해봤는데..
수행이 늘어날수록 메모리 사용량이 계속 늘어나기만 하는데..

<img width="156" alt="스크린샷 2023-03-26 17 07 42" src="https://user-images.githubusercontent.com/60438045/227765205-0bf14680-32c1-4e64-b303-be7b3246fb70.png">

`disposed` 될 때 프린트문을 넣어봤더니 제대로 되고 있거든..

이 정도는 허용 범위인건지..
실제로 어딘가에서 누수가 일어나고 있는건지 알아봐야할 것 같아

**Comment by EnnzoKim** [comment]
이거 혹시 이게 문제일 가능성은 있을까? 
Cell에 이벤트를 Rx로 처리할 때 발생되는 이슈에 대한 글이야
한번 읽어보고 의견 남겨줘! 

https://sueaty.tistory.com/158
